client/deployment/src/main/java/io/quarkiverse/openapi/generator/deployment/CommonItemConfig.java

```diff
@@ -110,4 +110,10 @@ public class CommonItemConfig {
      */
     @ConfigItem(name = "use-field-name-in-part-filename")
     public Optional<Boolean> useFieldNameInPartFilename;
+
+    /**
+     * Enable bean validation. If you set this to `true`, validation annotations are added to generated sources E.g. @Size.
```

Comment:
Markdown is not rendered in Javadoc. You need to use `{@code}` for that.
```suggestion
Enable bean validation. If you set this to {@code true}, validation annotations are added to generated sources E.g. {@code @Size}.
```